open-craft/xblock-poll
# [BB-3685] Fix CI Runs

**Description**

Fix the CI build by:

1. Removing the python 2.7 build 
2. Pinning [xblock-utils](https://github.com/edx/xblock-utils) to version 2.1.0

**Supporting information**
Jira Ticket: [BB-3685](https://tasks.opencraft.com/browse/BB-3685)
[xblock-utils 2.1.0](https://github.com/edx/xblock-utils/releases/tag/2.1.0)

**Testing instructions**

1. On this PR, confirm that the CI builds have passed
2. Run `make requirements` to install requirements. Run `make build_dummy_translations` to extract and compile dummy translations. Confirm that the name of the directories in `poll/public/js/translations/` are as case-senstive as the ones created `poll/translations/` i.e. `ko_KR` translates to `ko_KR` and not `ko_kr`

**Deadline**

End of sprint BB.242
